src/components/screens/HomeScreen/RenameAccountModal.tsx

```diff
@@ -85,7 +85,7 @@ const RenameAccountModal: React.FC<RenameAccountModalProps> = ({
             onPress={() => setModalVisible(false)}
             disabled={isRenamingAccount}
           >
-            {t("renameAccountModal.skip")}
+            {t("renameAccountModal.cancel")}
```

Comment:
nit: you can use the common keys so you don't have to add another key for this
```suggestion
            {t("common.cancel")}
```

Reply:
removed extra xancel and using common.cancel